### PR TITLE
fix: stop process only if the title matches exactly

### PR DESCRIPTION
**Pull request by plusmancn**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

`egg-scripts`

##### Description of change

Because of'exmaple'.inclues('exmap') === true，if egg-scripts <= 2.1.0 and you run `.. stop --title=exmap`，the process with 'title:example' will also be killed unexpectedly，see below
1. [/lib/cmd/stop.js#L40](https://github.com/eggjs/egg-scripts/blob/1a8ad1f3baab7f9a1238d239bb5189ab26572be2/lib/cmd/stop.js#L40),
2. [/lib/cmd/stop.js#L60](https://github.com/eggjs/egg-scripts/blob/1a8ad1f3baab7f9a1238d239bb5189ab26572be2/lib/cmd/stop.js#L60)

This PR make egg-scripts stop process only if the title matches exactly.

close https://github.com/eggjs/egg/issues/3313

### CODE REVIEW

**Comment by plusmancn** [comment]
it seems that the Appveyor failure not caused by this pr.

**Comment by atian25** [comment]
could you help to fix it at this pr

**Comment by plusmancn** [comment]
> could you help to fix it at this pr

I can have a try.

File: lib/cmd/start.js [hunk 1]
```diff
@@ -108,7 +109,8 @@ class StartCommand extends Command {
     env.HOME = HOME;
     env.NODE_ENV = 'production';
 
-    env.PATH = [
+    const platformPath = isWin ? 'Path' : 'PATH';
```
[the remark this answers is not in the document]
Comment (plusmancn) [reply lib/cmd/start.js]: it makes the env so big.

[the remark this answers is not in the document]
Comment (plusmancn) [reply lib/cmd/start.js]: When you start a new node process, node will read `env.Path` on windows, read `env.PATH` on Linux, but all will be mounted on `process.env.PATH`, the user can always access it by `process.env.PATH`.
In another case, the process has started, the user set `process.env.Path` or `process.env.PATH` at runtime, it's case-sensitive which will confuse the user.
For simple and robust, we just set both as the latest commit do.

**Pull request merged**